.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+> Provide a brief description about **why** this PR is necessary. Be sure to provide context.
+
+## Related issues and/or PRs
+
+> If this PR addresses or references any issues and/or other PRs, list them here. For example, "Fixes #123", "Related to #456", or "Addresses #789".
+
+## Changes made
+
+> Outline the specific changes made in this pull request. Include relevant details, such as added features, bug fixes, code refactoring, or improvements.
+
+## How these changes were tested
+
+> Describe the testing that you have performed to ensure the changes are working as expected. Include details about the test cases or scenarios covered.
+
+## Checklist
+
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have updated the documentation to reflect the changes.
+- [ ] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
+- [ ] Tests (unit, integration, etc.) have been added for the changes.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes in other PRs have been merged and published.
+
+## Additional notes (optional)
+
+> Provide any additional information or notes that may be relevant to the reviewers or stakeholders.
+
+## Release notes
```

Comment:
Just a question.

We don't have any release notes for this repository (for in-house docker image), but do we need to this section in the PR template in this repository?

Or, do you have any plan to create a release note for this in-house docker image in the future?

Reply:
Since this repo is in-house only, I'm OK with removing the `## Release note` section. If we decide that we need the section later, we can always re-add it.

Reply:
Yes. Removing the `## Release note` section is OK with me! It sounds good!